src/ccnl-pkt/src/ccnl-pkt-ccntlv.c

```diff
@@ -84,14 +84,23 @@ ccnl_ccntlv_getHdrLen(unsigned char *data, int len)
     return -1;
 }
 
-// parse TL (returned in typ and vallen) and adjust buf and len
+
+/**
+ * parse TL (returned in typ and vallen) and adjust buf and len
+ * @param buf allocated buffer in which the tlv should be opened
+ * @param len length of the buffer
+ * @param typ return value via pointer: type value of the tlv
+ * @param vallen return value via pointer: length value of the tlv
+ * @return 0 on success, -1 on failure.
+ */
```

Comment:
I appreciate your boyscout/documentation efforts, but I would put the documentation in the header file

Reply:
i agree